ui/components/SubMenu.tsx

```diff
@@ -19,7 +19,7 @@ const groupItems = ({ currentUser, groupSlug }) => {
   return [
     { label: "Overview", href: `/${groupSlug}` },
     // { label: "Realities", href: "/realities" },
-    { label: "Members", href: `/${groupSlug}/members`, admin: true },
+    { label: "Participants", href: `/${groupSlug}/participants`, admin: true },
```

Comment:
i think on the group level they should still be called Members, it's only in the round that they're called Participants